api/src/main/resources/marquez/db/migration/V23__index_transitioned_at_on_run_states.sql

```diff
@@ -0,0 +1,3 @@
+create index run_states_transitioned_at_index
+    on run_states (transitioned_at desc);
```

Comment:
Adding this index seems fine with me, @henneberger thoughts?

Reply:
Should this on be `runs`, not `run_states`?

Reply:
Oh, that is in the PR that I've been trying to get landed.

Reply:
Yeah, currently, the `transitioned_at` column is only on the `run_states` table.

Reply:
This index isn't good since we aren't trying to order the entire run state table by transition time.  Can we wait until that gets landed and have an index that does:
```
create index run_transitioned_at_index
    on run_states (namespace_name, job_name, transitioned_at desc);
    ```

Reply:
Sure, so remove the `transitioned_at` index for now? /cc @phixMe

Reply:
@wslulciuc @henneberger Sounds good, removing it.